### PR TITLE
vsc5 In Class One

### PR DESCRIPTION
**Pull request by vsc5**
The current spacing is weird, will try to fix. Just testing a pull request first.

### CODE REVIEW

**Comment by vsc5** [comment]
I fixed the spacing

File: index.html [hunk 1]
```diff
@@ -1,11 +1,97 @@
...
+    <meta content="Creator: vsc5">
     <title>In class Exercise Numero Uno</title>
     <link rel="stylesheet" href="./css/icOne.css">
   </head>
-  <body>
 
+  <body>
+    <header>
+      <h1>
+        <a href="">
```
Comment (mattbowytz) [inline index.html]: try not to leave empty attributes, for `href`s just use a `#` of something to hold the place

File: index.html [hunk 2]
```diff
@@ -1,11 +1,97 @@
...
+          </li>
+          <li>
+            <a href="http://laravegas.tumblr.com">Blog</a>
+         </li>
+        </ul>
+      </nav>
+    </header>
+    <div class="pt">
+      <span style="color:#0912F9;">Fill this out... Or else.</span>
+      <form method="post" action="/" name="formtastic" onsubmit="return validation()">
```
Comment (mattbowytz) [inline index.html]: fancy!

File: index.html [hunk 3]
```diff
@@ -1,11 +1,97 @@
...
+    </div>
+    <footer>
+      &copy;2016 Vivien Chang &vellip; <a href="">Contact</a>
+    </footer>
   </body>
-</html>
\ No newline at end of file
+
+  <script>
+    function validation() {
```
Comment (mattbowytz) [inline index.html]: oh snap!!!

**Comment by mattbowytz** [comment]
👍

**Comment by matty-digital** [comment]
disregard comment about deleting branch, I was in the wrong account

**Pull request closed without merging**